contrib/storage-hive/core/src/test/java/org/apache/drill/exec/hive/TestHiveStorage.java

```diff
@@ -510,6 +517,38 @@ public void testIgnoreSkipHeaderFooterForSequencefile() throws Exception {
         .go();
   }
 
+  @Test
```

Comment:
Seems to be no test for the Decimal precision mapping. That may be why the issue mentioned in the code slipped through: there is no unit test to catch it...